tweets/2020/semantic-release-helm.tweet

```diff
@@ -0,0 +1,3 @@
+📯 Manage your Helm charts with semantic-release:
+
+https://www.npmjs.com/package/semantic-release-helm
```

Comment:
I think it would be good to mention kubernetes. How about this:

```
📯 Kubernetes friends: you can now manage your @helmpack charts with semantic-release:

https://www.npmjs.com/package/semantic-release-helm
```

Do you have a twitter account yourself?

Reply:
Changed it. I have no twitter account.